index.html

```diff
@@ -4,18 +4,169 @@
 <head>
     <meta charset="UTF-8">
 
-    <title>Untitled</title>
+    <title>Smashing Magazine - HeatMap</title>
 
     <link rel="stylesheet" href="css/styles.css">
 
+    <link href="https://fonts.googleapis.com/css?family=Fira+Sans:200,500,700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Playfair+Display:400,700&display=swap" rel="stylesheet">
+
 </head>
 
 <body>
     <header>
-
+        <div class="header-cont bordered"></div>
+        <div class="header-cont bordered"></div>
+        <div class="header-cont bordered"></div>
+        <div class="header-cont bordered"></div>
+        <div class="header-cont "></div>
     </header>
 
     <main>
+        <div class="title-div">
+            <h2>Don't Miss These Latest <br>Articles</h2>
+        </div>
+        <div class="article-div">
+            <div class="text-div">
+                <h3>Daniel Matheus <i>wrote</i></h3>
+
+                <h1>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</h1>
+
+                <p>Leave a lorem ipsum</p>
+                <br><br>
+                <p>#lorem #ipsum</p>
+            </div>
+        </div>
+        <div class="article-div">
+            <div class="text-div">
+                <h3>Daniel Matheus <i>wrote</i></h3>
+
+                <h1>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</h1>
+
+                <p>Leave a lorem ipsum</p>
+                <br><br>
+                <p>#lorem #ipsum</p>
+            </div>
+        </div>
+        <div class="article-div">
+            <div class="text-div">
+                <h3>Daniel Matheus <i>wrote</i></h3>
+
+                <h1>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</h1>
+
+                <p>Leave a lorem ipsum</p>
+                <br><br>
+                <p>#lorem #ipsum</p>
+            </div>
+        </div>
+
+        <div class="membership-cont">
+            <div class="membership-ad">
+                <p>All you want is a <b>fast, secure lorem ipsum</b>? Dolor sit amet, consectetur adipiscing elit. In
+                    faucibus sit amet tellus nec efficitur.
+                    Maecenas consequat et libero at interdum. Ut pellentesque eget mi a finibus. Phasellus ac tincidunt
+                    ante. Suspendisse potenti. Mauris in tellus magna. Morbi quis magna elit. </p>
+                <ul>
+                    <li>Lorem ipsum dolor sit amet, consectetur <b>adipiscing elit.</b></li>
+                    <li>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</li>
+                    <li>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</li>
+                </ul>
+            </div>
+            <div class="membership-img">
+
+            </div>
+        </div>
+
+        <div>
```

Comment:
This one also should be a `<section>` tag with its correspondent `<article>` tags, these apart from the requirements stated before should have these characteristics:
* [ ] Three articles with a width of `33%`
**So they should span across the width of the site I know on the original site theres some white space left on bigger screen sizes but for the scope of this project is is required to be that way**